vicar/vicarlabel.py

```
@@ -2273,73 +2280,78 @@ def read_label(source, _extra=False):
-            # Figure out the distance to the EOL label
-            recsize = ldict['RECSIZE']
-            nlb = ldict.get('NLB', 0)
-            # N2*N3 is simpler but there are files where these values aren't right
-            if ldict['ORG'] == 'BIP':           # pragma: no cover
-                data_recs = ldict['NL'] * ldict['NS']
-            else:
-                data_recs = ldict['NL'] * ldict['NB']
-            skip = lblsize + recsize * (nlb + data_recs)
-            f.seek(skip)
+    @staticmethod
+    def _read_label(f, filepath, _extra):
```

Ruff (0.14.5)

Missing return type annotation for staticmethod `_read_label`

(ANN205)